src/components/ModifyAnnouncementModal.js

```diff
@@ -50,7 +50,6 @@ const ModifyAnnouncementModal = ({
     tmp["content"] = draftToHtml(convertToRaw(editorState.getCurrentContent()));
     tmp["top"] = form.current.elements.top.checked ? 1 : 0;
     tmp["posted_by"] = account.id;
-    tmp["timestamp"] = new Date();
```

Comment:
It might be better to use server timestamp rather than client timestamp. I think that firebase provides such service.

Reply:
Indeed, we can address this issue in next pull request